docs/learn/learn-nomination-pools.md

```diff
@@ -0,0 +1,154 @@
+---
+id: learn-nomination-pools
+title: Nomination Pools
+sidebar_label: Nomination Pools
+description: Learn about Nomination Pools and their features
+keyword: [nominate, nominator, stake, staking, pools]
+slug: ../learn-nomination-pools
+---
+
```

Comment:
banner about deployment plan would be good: currently westend, kusama soon, polkadot afterwards.